app/src/shared.js

```diff
@@ -125,3 +125,17 @@ export function tradeCanBeRefunded(tradeInfo, walletAddr) {
   const { trade } = tradeInfo
   return trade.state === 'escrow_funded' && tradeInfo.expired && trade.sender === walletAddr
 }
+
+export function removeTelegramURLPrefix(telegram) {
```

Comment:
This can be simplified to something like:
```
const search = 't.me/'
const start = telegram.indexOf(search) + search.length
return telegram.includes(search) : telegram.substr(start) ? telegram
```

Reply:
Besides the comment above, LGTM. All tests are running and the feature works well in the browser!

Reply:
Addressed in [bbf4d3b](https://github.com/Local-Money/localmoney/pull/82/commits/bbf4d3b821f972b8f391dda5b0b3e2c4050fab58)